dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java

```diff
@@ -23,13 +27,12 @@
 import nl.tudelft.dnainator.javafx.widgets.animations.TransitionAnimation.Position;
 import nl.tudelft.dnainator.javafx.widgets.dialogs.ExceptionDialog;
 import nl.tudelft.dnainator.javafx.widgets.dialogs.ProgressDialog;
-import org.neo4j.io.fs.FileUtils;
```

Comment:
The frontend now does not depend on Neo4j anymore. As such, this dependency should be removed from the parent pom and placed in `dnainator-javafx/pom.xml` instead.